flocker/ca/_script.py

```diff
@@ -226,9 +226,11 @@ class ControlCertificateOptions(PrettyOptions):
     (defaults to current working directory).
     """
 
-    synopsis = "[options]"
+    synopsis = "[options] --hostname <hostname>"
```

Comment:
`[options]` comes before `--hostname <hostname>`.  Also this tool takes other `--options` that aren't mentioned here...  And `--hostname` is optional?  Or mandatory?